src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -185,17 +188,25 @@ public async Task<JObject> Resolve(string varName, CancellationToken token)
             if (scopeCache.ObjectFields.TryGetValue(varName, out JObject valueRet))
                 return await GetValueFromObject(valueRet, token);
 
-            string[] parts = varName.Split(".");
+            string[] parts = varName.Split(".", StringSplitOptions.TrimEntries);
```

Comment:
E.g. `DebuggerTests.EvaluateOnCallFrameTests.EvaluateMethodLocals` or `DebuggerTests.EvaluateOnCallFrameTests.AsyncLocalsInContinueWithBlock` are testing whitespaces. But in this very place it was not necessary because we were Trimming each `part` separately when processing it. I decided that instead, I will remove Trimming from `ResolveAsLocalOrThisMember` and `ResolveAsInstanceMember` and do it only once here, where you added this comment.